after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
```

Comment:
How do you think I should handle this? I could:
1. Revert back and just add the code to enable functionality
2. Come up with an alternate way to ensure it's both a Gradle project and an Android project

The lag that is there as a result of opening an XML file seems odd, but I think that can be eliminated if we grab the classpaths from Javacomplete instead of generating them on our own.